projects/novo-elements/src/elements/date-picker/DatePicker.ts

```diff
@@ -130,7 +130,7 @@ export type rangeSelectModes = 'startDate' | 'endDate';
                             endfill: isEndFill(range, day.date, selected, selected2),
                             'selecting-range': isSelectingRange(range, day.date, selected, selected2, hoverDay, rangeSelectMode, weekRangeSelect)
                            }" (mouseover)="rangeHover($event, day)" [attr.data-automation-id]="day.number">
-                            <button class="day" [attr.data-automation-id]="day.number" [disabled]="isDisabled(day.date, start, end)" (click)="select($event, day, true)">{{day.number}}</button>
+                            <button [title]="isDisabled(day.date, start, end) ? customTooltip : ''" class="day" [attr.data-automation-id]="day.number" [disabled]="isDisabled(day.date, start, end)" (click)="select($event, day, true)">{{day.number}}</button>
```

Comment:
This MR is not complete until there are examples of this in the Novo Elements Demo App. You will need to add at least one example to the current Date Picker Element page to show the min/max dates in action. You will want to provide pickers to specify the min/max dates and offsets, and allow the user to change them dynamically to reflect in the date picker.

Reply:
done attached a demo video